### PR TITLE
`signin` is not fired when logging in via token

### PR DESCRIPTION
**Pull request by sohini-roy**
closes hoodiehq/camp#120

### CODE REVIEW

File: lib/sign-in.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function signIn (state, options) {
         }
 
         // If the username hasn’t changed, emit 'reauthenticate' instead of 'signin'
-        var emitEvent = cache.username === options.username
+        var emitEvent = cache.username === data.account.username
```
Comment (gr2m) [inline lib/sign-in.js]: this fixes the problem 👍

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "standard": "^10.0.1",
     "tap-min": "^1.0.0",
     "tap-spec": "^4.1.0",
-    "tape": "^4.2.0",
+    "tape": "^4.7.0",
```
Comment (gr2m) [inline package.json]: please revert this change, otherwise it’s good to go :)

**Comment by rmehner** [comment]
Thanks @sohini-roy 👍
